ksqldb-api-client/src/main/java/io/confluent/ksql/api/client/impl/ClientImpl.java

```diff
@@ -196,7 +196,7 @@ public BatchedQueryResult executeQuery(
 
     makePostRequest(
         CLOSE_QUERY_ENDPOINT,
-        new JsonObject().put("queryId", queryId),
+        new JsonObject().put("queryId", queryId).put("sessionVariables", sessionVariables),
```

Comment:
Does this work? Does the close query endpoint actually support variable substitution?